recipes/pyjion/meta.yaml

```
@@ -0,0 +1,68 @@
+{% set name = "pyjion" %}
+{% set version = "1.2.6" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  git_url: https://github.com/tonybaloney/Pyjion.git
+  git_tag: {{ version }}
+
+
+build:
+  skip: True  # [py < 310]
+  number: 0
+  script: |
+    git submodule update --init --recursive
+    {{ PYTHON }} -m pip install . -vv
+  entry_points:
+    - pyjion = pyjion.__main__:main
+
+requirements:
+  build:
+    - cmake
+    - dotnet
+    - git
+    - ninja
+  host:
+    - python
```

For testing you can add this to `host` an `run` (remove it before merging):
```suggestion
    - python 3.10.* *_cpython
```